### ISSUE TITLE
Dataset

### ISSUE THREAD

**Issue by Zhengxl25** [shvdiwnkozbw/Multi-Source-Sound-Localization#9]
Hi shvdiwnkozbw！
Thank you so much for sharing your code. Can you provide the AudioSet Instrument dataset?Looking forward to your reply!

**Comment by shvdiwnkozbw** [comment]
Sorry, the AudioSet Instrument dataset is used in previous server, no access to the data itself currently.
But it can be filtered by the audio category labels in original AudioSet dataset.

**Comment by Devin-Pi** [comment]
Hi shvdiwnkozbw！Thanks for sharing your work.
I would like to ask the "pairs" in the file "./prepare_data/data.py" is the "./utils/dataset.pkl". Or should I prepare the list representing audio-visual pairs index by myself according to my datasets?
Looking forward to your reply! Thank you.

**Comment by shvdiwnkozbw** [comment]
Yes, the pairs depend on the data you use for training. You are expected to generate it according to your dataset.

**Comment by Devin-Pi** [comment]
> Yes, the pairs depend on the data you use for training. You are expected to generate it according to your dataset.

Wow, thank you, shvdiwnkozbw!
What's more, I just found a problem happened in "./prepare/data.py", as shown below.
![image](https://user-images.githubusercontent.com/89199086/211572164-0ff630c8-4ef4-46a0-8781-2ab9d0490c2d.png)
Based on Google, the reason for this problem is that the length for those "audios" are different. So, have you encountered this problem and how did you solve this? And can I solve this problem according to this method shown in the link https://stackoverflow.com/questions/52841335/how-can-i-pad-wav-file-to-specific-length/70948409#70948409 .
Thank you! Looking forward to your reply.

**Comment by shvdiwnkozbw** [comment]
I have not encountered this problem. It might be due to the specific data you use. I think it should be fine to solve the problem referring to this link.

**Comment by Devin-Pi** [comment]
> I have not encountered this problem. It might be due to the specific data you use. I think it should be fine to solve the problem referring to this link.

Thanks for your response!
